tests/by-util/test_du.rs

```diff
@@ -172,3 +172,23 @@ fn test_du_h_flag_empty_file() {
     assert_eq!(result.stderr, "");
     assert_eq!(result.stdout, "0\tempty.txt\n");
 }
+
+// I don't have touch on windows so I can't run this test
+// if we port touch to windows then we can enable this test on windows
```

Comment:
Apparently if I build with `--features windows` we have `touch`. So I probably can update this test.